tgui/packages/tgui/interfaces/PowerTransmissionLaser.js

```diff
@@ -0,0 +1,216 @@
+/**
+ * @file
+ * @copyright 2020
+ * @author Sovexe (https://github.com/Sovexe)
+ * @license ISC
+ */
+
+import { useBackend } from '../backend';
+import { Box, Button, ColorBox, Divider, Knob, LabeledList, NoticeBox, ProgressBar, Section } from '../components';
+import { formatMoney, formatPower, formatSiUnit } from '../format';
+import { Window } from '../layouts';
+
+export const PowerTransmissionLaser = (props, context) => {
+  const { data } = useBackend(context);
+  const {
+    lifetimeEarnings,
+    name = 'Power Transmission Laser',
+  } = data;
+  return (
+    <Window
+      title={name}
+      width="310"
+      height="485">
+      <Window.Content>
+        <Status />
+        <InputControls />
+        <OutputControls />
+        <NoticeBox success>
+          Earned Credits : {formatMoney(lifetimeEarnings)}
```

Comment:
does this not put commas between the 3-unit-seperation? doesn't look like it on the image

Reply:
It is just spaces, we can change it if you want